registration/forms.py

```diff
@@ -355,6 +385,41 @@ def update_user(self, application, commit=True):
                 application.user.profile.save()
                 application.save()
 
+        if self.restart_fields_changed():
+            messages.add_message(self.request, messages.INFO,
+                                 "Thank you for submitting these changes - we will rebuild your instance to "
+                                 "apply them, and email you to confirm once it is up to date.")
+
+            if settings.VARIABLES_NOTIFICATION_EMAIL:
+                subject = '{prefix} Update required at instance {name}'.format(
+                    prefix=settings.EMAIL_SUBJECT_PREFIX,
+                    name=application.subdomain,
+                )
+                text = ("Instance {instance_id} at {domain} requires a redeployment "
+                        "because some values were changed through the "
+                        "registration form.\n"
+                        "\n"
+                        "The new values are:\n"
+                        "- main color: {main_color}\n"
+                        "- link color: {link_color}\n"
+                        "- primary background color: {bg_color_1}\n"
+                        "- secondary background color: {bg_color_2}\n"
+                        "- logo: {logo}\n"
+                        "- favicon: {favicon}\n"
+                       ).format(
```

Comment:
@antoviaque, could you take a look at the content of this email to see if it'll be sufficient for the betatest workflow needs?

Reply:
Note that I updated it (I removed "[OpenCraft]: " from the subject).
Check also the sending address (now: `settings.DEFAULT_FROM_EMAIL`).

Reply:
@bdero @clemente That looks good yes. When we receive this on help@, the only action the  @aliciaopencraft17 would need to do then would be to start a new VM and activate it, right? Could you write down exactly these steps, so she can keep this in one of her recurring tasks?

Reply:
@clemente Daniel, I created https://tasks.opencraft.com/browse/ADMIN-58 for this. Thank you in advance for writing down these steps - please keep in mind to "dumb" down the language and explain acronyms for me to try and understand it from a non-dev mind :) I might still have questions but hopefully this will reduce them. Thank you!
cc @antoviaque

Reply:
@aliciaopencraft17 Thank you!

Reply:
@antoviaque @aliciaopencraft17 Yes it will be like that, Alicia will only need to click the „Launch new appserver“ button. I added instructions to ADMIN-58, including manual checks that might be necessary if we need to debug things.

Reply:
@clemente Thank you!